space-cli/cmd/modules/addons/database.go

```diff
@@ -1,379 +1,51 @@
-					"rule": map[string]interface{}{
-						"rule": "allow",
-					},
-					"args": []string{},
-				},
-			}
-			if err := operations.ApplySpec(token, account, v); err != nil {
-				logrus.Warningln("Couldn't add default prepared query rules", nil)
-			}
-
-			utils.LogInfo("Successfully added database to Space Cloud config.")
-			return
-		}
-	}
-}
-
-func removeDatabase(alias string) error {
-	clusterName := viper.GetString("cluster-name")
-	project, check := utils.GetProjectID()
-	if !check {
-		_ = utils.LogError("Project not specified in flag", nil)
-		return nil
-	}
-	autoRemove := viper.GetBool("auto-remove")
-
-	// change selected account according to cluster name provided
-	if err := utils.ChangeSelectedAccount(clusterName); err != nil {
-		return utils.LogError("Could not set selected account ", err)
-	}
-
-	ctx := context.Background()
-
-	// Create a docker client
-	docker, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
-	if err != nil {
-		return utils.LogError("Unable to initialize docker client", err)
-	}
-
-	// Check if a database container already exist
-	argsDB := filters.Arg("name", utils.GetDatabaseContainerName(clusterName, alias))
-	argsNetwork := filters.Arg("network", utils.GetNetworkName(clusterName))
-	containers, err := docker.ContainerList(ctx, types.ContainerListOptions{Filters: filters.NewArgs(argsNetwork, argsDB)})
-	if err != nil {
-		return utils.LogError("Unable to check if database already exists", err)
-	}
-	if len(containers) == 0 {
-		utils.LogInfo(fmt.Sprintf("Database (%s) not found.", alias))
-		return nil
-	}
-
-	if autoRemove {
-		if project == "" {
-			return utils.LogError(`Please provide project id through "--project" flag`, nil)
-		}
-
-		// remove database from space-cloud
-		if err := deleteDatabaseConfig(project, alias); err != nil {
-			return err
-		}
-	}
-
-	// Get the hosts file
-	hosts, err := txeh.NewHosts(&txeh.HostsConfig{ReadFilePath: utils.GetSpaceCloudHostsFilePath(clusterName), WriteFilePath: utils.GetSpaceCloudHostsFilePath(clusterName)})
-	if err != nil {
-		return utils.LogError("Unable to open hosts file", err)
-	}
-
-	for _, c := range containers {
-		hostName := utils.GetServiceDomain("db", alias)
-
-		// Remove the domain from the hosts file
-		hosts.RemoveHost(hostName)
-
-		// remove the container from host machine
-		if err := docker.ContainerRemove(ctx, c.ID, types.ContainerRemoveOptions{Force: true}); err != nil {
-			return utils.LogError(fmt.Sprintf("Unable to remove container %s", c.ID), err)
-		}
+		return fmt.Errorf("unkown database (%s) provided as argument", chartReleaseName)
 	}
 
-	// Save the hosts file
-	if err := hosts.Save(); err != nil {
-		return utils.LogError("Could not save hosts file after updating add on containers", err)
-	}
-
-	utils.LogInfo(fmt.Sprintf("Removed database (%s)", alias))
-
-	return nil
+	_, err = utils.HelmInstall(chartReleaseName, chartLocation, downloadURL, helmDatabaseNamespace, valuesFileObj)
```

Comment:
At least wait for the pods to come up and print the domain name the user can use to connect to the database since you are not auto applying.

Do a watch on label `name` in the db namespace and wait for atleast 3mins for the pods to come up